x-pack/plugins/enterprise_search/public/applications/workplace_search/cypress/integration/overview.spec.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { login } from '../support/commands';
+
+context('Overview', () => {
+  beforeEach(() => {
+    login();
+  });
+
+  it('renders', () => {
+    cy.contains('Workplace Search');
```

Comment:
Heads up @scottybollinger and @yakhinvadim - unfortunately I'm not 100% sure how to solve a few issues specific to your current ent-search E2E tests:

- Running `rake` commands to populate connectors/sources is unfortunately no longer a thing, since we don't have those scripts in Kibana
  - However, since these are now meant to be nightly tests, one option here is you could point your `baseUrl` at the remote instance Adam has set up for WS's nightly backend e2e tests, which I believe already has specific sources & connector data prepopulated 🤔 You'd have to confer with Adam, and maybe Sean (who was on a call with me and Tarek when we brainstormed a little bit about this) to set this up though

- Basic vs. platinum tests - IIRC you have them split up into separate test suites in ent-search, which would be slightly more painful in Kibana but still doable.
  - One alternative to consider would be splitting your tests up into folders instead, e.g. `workplace_search/cypress/integrations/basic` & `workplace_search/cypress/integrations/platinum` and configuring your `cypress.json` to run the basic test folder first, then upgrade to a trial via Kibana's license management UI, then run platinum tests (you can order tests via the `testFiles` config).

Reply:
Sounds good, thanks, Constance!